modules/integration/src/test/scala/scala/cli/integration/FixBuiltInRulesTestDefinitions.scala

```diff
@@ -425,4 +419,32 @@ trait FixBuiltInRulesTestDefinitions { _: FixTestDefinitions =>
       )
     }
   }
+
+  if (!Properties.isWin) // TODO: fix this test for Windows CI
```

Comment:
@tgodzik no idea why this is failing on Windows CI, I'll investigate as a follow-up